auslib/blobs/base.py

```diff
@@ -123,3 +123,6 @@ def getFooterXML(self, updateQuery, update_type, whitelistedDomains, specialForc
 
     def getInnerXML(self, updateQuery, update_type, whitelistedDomains, specialForceHosts):
         raise NotImplementedError()
+
+    def containsForbiddenDomain(self, product, whitelistedDomains):
+        return False
```

Comment:
I also wonder if we should be calling this from .validate() instead of requiring callers to make a separate call for it. It's another thing that would make it harder for to make mistakes later. What do you think about that?